### PR TITLE
fix: api reports styling

### PR DESCRIPTION
**Pull request by Ziinc**
This PR adds in the following changes:
- reduced x,b padding for routes tables
- add collapsible toggle
- add "Open in Logs Explorer" button

https://github.com/supabase/supabase/assets/22714384/9ea49cbe-e216-4c9c-91d2-d33250b7cf9d

### CODE REVIEW

File: studio/components/interfaces/Reports/renderers/ApiRenderers.tsx [hunk 1]
```diff
@@ -105,42 +107,89 @@ export const TopApiRoutesRenderer = (
...
+                query: {
+                  q: props.params?.sql,
+                  its: props.params!.iso_timestamp_start,
+                  ite: props.params!.iso_timestamp_end,
+                },
+              })
+            }}
+          >
+            Open in Logs Explorer
+          </Button>
```
Comment (alaister) [inline studio/components/interfaces/Reports/renderers/ApiRenderers.tsx:189]: Small styling nit:
Screenshot 2023-10-02 at 14 53 12
the padding around this button looks incosistent

**Review by alaister — CHANGES_REQUESTED** [review]
Looks great! Only one small nit:

File: studio/components/interfaces/Reports/renderers/ApiRenderers.tsx (hunk 1 above)
Comment (joshenlim) [reply studio/components/interfaces/Reports/renderers/ApiRenderers.tsx:189]: yup - a little more obvious in dark mode 
image

**Comment by joshenlim** [comment]
also another styling feedback - should we just remove the padding here and border-radius? so there's no gap between the table and the parent container. but otherwise lgtm too
image

**Comment by Ziinc** [comment]
Screenshot 2023-10-17 at 1 23 11 AM
Screenshot 2023-10-17 at 1 23 06 AM
styling tweaks are in 😄

**Review by alaister — APPROVED** [review]
LGTM!

**Pull request merged**